carp.studies.core/src/commonTest/kotlin/dk/cachet/carp/studies/StudiesCodeSamples.kt

```diff
@@ -58,8 +58,8 @@ class StudiesCodeSamples
         // Once the study is live, you can 'deploy' it to participant's devices. They will be invited.
         if ( studyStatus.canDeployToParticipants )
         {
-            // Create a 'participant group' with a single participant, using the "Patient's phone".
-            val participation = AssignParticipantDevices( participant.id, setOf( patientPhone.roleName ) )
+            // Create a 'participant group' with a single participant, with no specific participant role assignment.
```

Comment:
Maybe emphasize that the example protocol contains the primary smartphone device already and how this links to the participation created here.